BuildLoader.py

```diff
@@ -361,6 +361,8 @@ def update_fit_table (self):
                     u_code_images.append(ucode_offset)
                     num_fit_entries += 1
                     if (hasattr (self._board, "UCODE_SLOT_SIZE")):
+                        if ucode_hdr.total_size > self._board.UCODE_SLOT_SIZE:
+                            raise Exception ('UCODE total size (0x%08X) greater than UCODE slot size' % (ucode_hdr.total_size, self._board.UCODE_SLOT_SIZE))
```

Comment:
ucode_hdr.total_size should be same with self._board.UCODE_SLOT_SIZE, right?

Reply:
The uCode header is pulled directly from a uCode file -- we don't have direct control over it. It is the size of the actual uCode file, without factoring in padding.

Reply:
It makes sense if it is not padded yet.